specification/columns/skumetername.md

```diff
@@ -0,0 +1,42 @@
+# SKU Meter Name
+
+The SKU Meter Name describes the capability being metered or measured by a particular SKU in a charge.
+
+Providers often have billing models in which multiple SKUs exist for a given service to describe and bill for different capabilities for that service. For example, an object storage service may have separate SKUs for capabilities such as object storage, API requests, data transfer, encryption, object management. This field helps practitioners understand which capabilities are being metered by the different SKUs that appear in a FOCUS dataset.
+
+The SkuMeterName column adheres to the following requirements:
```

Comment:
In SkuPriceDetails, we included the following requirement:

> Individual properties (key-value pairs) SHOULD NOT be modified for a given SkuPriceId and SHOULD remain consistent over time.

Would it be helpful to include something similar here as well? For instance:

> SkuMeterName SHOULD NOT be modified for a given SkuId and SHOULD remain consistent over time.

Reply:
Makes sense - added.